src/experimental/players/DesktopReskinnedArticleVideoPlayer/DesktopReskinnedArticleVideoPlayer.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useRef, useState, useEffect } from 'react';
+import WDSVariables from '@fandom-frontend/design-system/dist/variables.json';
+import styled, { css } from 'styled-components';
+import useOnScreen from 'utils/useOnScreen';
+import PlayerWrapper from 'jwplayer/players/shared/PlayerWrapper';
+import { DesktopArticleVideoPlayerProps } from 'jwplayer/types';
+import Attribution from 'jwplayer/players/DesktopArticleVideoPlayer/Attribution';
+import { getArticleVideoConfig } from 'jwplayer/utils/articleVideo/articleVideoConfig';
+import articlePlayerOnReady from 'jwplayer/utils/articleVideo/articlePlayerOnReady';
+import JwPlayerWrapper from 'jwplayer/players/shared/JwPlayerWrapper';
+import DesktopReskinnedArticleVideoPlayerOverlay from 'experimental/players/DesktopReskinnedArticleVideoPlayer/DesktopReskinnedArticleVideoPlayerOverlay';
+import DesktopScrollVideoTopContent from 'experimental/players/DesktopReskinnedArticleVideoPlayer/DesktopScrollVideoTopContent';
+
+const DesktopArticleVideoTopPlaceholder = styled.div`
+	z-index: ${Number(WDSVariables.z2) + 2};
+	position: absolute;
+	width: 100%;
+	padding-top: 56.25%;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
+`;
+
+interface DesktopArticleVideoWrapperProps {
+	isScrollPlayer: boolean;
+	right?: number;
+	width?: number;
+}
+
+const DesktopArticleVideoWrapper = styled.div<DesktopArticleVideoWrapperProps>`
+	height: max-content;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+
+	${(props) =>
+		props.isScrollPlayer
+			? css`
+					right: 18px;
+					bottom: 45px;
+					width: 300px;
+					position: fixed;
+			  `
+			: css`
+					position: absolute;
+					bottom: 0;
+					right: 0;
+					top: 0;
+					left: 0;
+			  `}
+`;
+
+const DesktopReskinnedVideoContentContainer = styled.div`
+	position: relative;
+	flex-grow: 2;
+`;
+
+const DesktopReskinnedArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlayerProps> = ({ videoDetails }) => {
+	const placeholderRef = useRef<HTMLDivElement>(null);
+	/* TODO: FIX THIS - This is used to test the player locally ONLY */
```

Comment:
Adding this comment to make sure we don't leave `adComplete = true` when deploying to sandbox.